tests/test_precision.py

```diff
@@ -146,20 +147,70 @@ def test_snippets():
         cmp_regimes,
     ) = stumpy.snippets(T, m, k, s=s, mpdist_T_subseq_isconstant=isconstant_custom_func)
 
-    npt.assert_almost_equal(
-        ref_snippets, cmp_snippets, decimal=config.STUMPY_TEST_PRECISION
-    )
-    npt.assert_almost_equal(
-        ref_indices, cmp_indices, decimal=config.STUMPY_TEST_PRECISION
+    # Revise fastmath flag, recompile, and re-calculate snippets,
+    # and then revert the changes
+    config.STUMPY_FASTMATH_FLAGS = {"nsz", "arcp", "contract", "afn"}
+    core._calculate_squared_distance.targetoptions["fastmath"] = (
+        config.STUMPY_FASTMATH_FLAGS
     )
-    npt.assert_almost_equal(
-        ref_profiles, cmp_profiles, decimal=config.STUMPY_TEST_PRECISION
-    )
-    npt.assert_almost_equal(
-        ref_fractions, cmp_fractions, decimal=config.STUMPY_TEST_PRECISION
+    njit_funcs = cache.get_njit_funcs()
+    for module_name, func_name in njit_funcs:
```

Comment:
This for-loop is repeated a few lines below after reseting the config variables. So, better to wrap a function around it and put it in `stumpy/cache.py` as suggested by @seanlaw.